frontends/precinct-scanner/src/app_tally_report_paths.test.tsx

```diff
@@ -130,387 +117,165 @@ function expectContestResultsInReport(
   }
 }
 
-test('expected tally reports are printed for a primary election with all precinct', async () => {
+beforeEach(() => {
+  jest.useFakeTimers();
+  fetchMock.reset();
+  fetchMock
+    .get('/machine-config', { body: getMachineConfigBody })
+    .get('/precinct-scanner/config/testMode', {
+      body: getTestModeConfigTrueResponseBody,
+    })
+    .get('/precinct-scanner/config/markThresholdOverrides', {
+      body: getMarkThresholdOverridesConfigNoMarkThresholdOverridesResponseBody,
+    });
+
+  const kiosk = fakeKiosk();
+  kiosk.getUsbDrives.mockResolvedValue([fakeUsbDrive()]);
+  kiosk.writeFile.mockResolvedValue(
+    fakeFileWriter() as unknown as ReturnType<KioskBrowser.Kiosk['writeFile']>
+  );
+  window.kiosk = kiosk;
+});
+
+test('printing: polls open, All Precincts, primary election + check additional report', async () => {
```

Comment:
This file has been largely rewritten and I recommend not trying to parse the diff.